reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md

```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
-### Example 15
+This example shows how to sort performance counter data. The example finds the processes on the
+computer that are using the most processor time during the sample.
+
+```powershell
+$Procs = Get-Counter -Counter "\Process(*)\% Processor Time"
+$Procs.CounterSamples | Sort-Object -Property CookedValue -Descending |
+   Format-Table -Property Path, InstanceName, CookedValue -AutoSize
 ```
-PS C:\> Get-Counter -Counter "\Processor(_Total)\% Processor Time" -Continuous
+
+```Output
+Path                                                         InstanceName             CookedValue
+----                                                         ------------             -----------
+\\Computer01\process(_total)\% processor time                _total              395.464129650573
+\\Computer01\process(idle)\% processor time                  idle                389.356575524695
+\\Computer01\process(mssense)\% processor time               mssense             3.05377706293879
+\\Computer01\process(csrss#1)\% processor time               csrss               1.52688853146939
+\\Computer01\process(microsoftedgecp#10)\% processor time    microsoftedgecp     1.52688853146939
+\\Computer01\process(runtimebroker#5)\% processor time       runtimebroker                      0
+\\Computer01\process(settingsynchost)\% processor time       settingsynchost                    0
+\\Computer01\process(microsoftedgecp#16)\% processor time    microsoftedgecp                    0
 ```
 
-This command gets a series of samples of the Processor\% Processor Time counter at the default one second interval.
-To stop the command, press CTRL + C.
+`Get-Counter` uses the **Counter** parameter to specify the `\Process(*)\% Processor Time` counter
+for all the processes on the local computer. The result is stored in the `$Procs` variable. The
+`$Procs` variable with the **CounterSamples** property sends the **PerformanceCounterSample**
+objects down the pipeline. `Sort-Object` uses the **Property** parameter to sort the objects by
+**CookedValue** in **Descending** order. `Format-Table` uses the **Property** parameter to select
+the columns for the output. The **AutoSize** parameter adjusts the column widths to minimize
+truncation.
+
 ## PARAMETERS
 
 ### -ComputerName
-Gets data from the specified computers.
-Type the NetBIOS name, an Internet Protocol (IP) address, or the fully qualified domain names of the computers.
-The default value is the local computer.
 
-Note:  Get-Counter does not rely on Windows PowerShell remoting.
-You can use the ComputerName parameter of Get-Counter even if your computer is not configured for remoting in Windows PowerShell.
+Specifies one computer name or a comma-separated array of **remote** computer names. Use the NetBIOS
+name, an IP address, or the computer's fully qualified domain name. **ComputerName** accepts strings
+from variables or values from a text file.
```

Comment:
Values from a text file? How? Strike the whole sentence. Not needed.

Reply:
resolved.